specs/sharding/beacon-chain.md

```diff
@@ -99,6 +101,14 @@ The following values are (non-configurable) constants used throughout the specif
 | `DOMAIN_SHARD_PROPOSER` | `DomainType('0x80000000')` |
 | `DOMAIN_SHARD_COMMITTEE` | `DomainType('0x81000000')` |
 
+### Shard Header Status
+
+| Name | Value | Notes |
+| - | - | - |
+| `UNCONFIRMED_SHARD_DATA` | `0` | Unconfirmed, nullified after confirmation time elapses |
+| `CONFIRMED_SHARD_DATA` | `1` | Confirmed, reduced to just the commitment |
+| `PENDING_SHARD_DATA` | `2` | Pending, a list of competing headers |
```

Comment:
Does it makes sense to somehow prefix those constants to group them? 
May be `SHARD_HEADER_STATUS_` or probably some shorter prefix?

Reply:
Formatting it as `SHARD_WORK_{status here}`, I want to avoid extra long constant names.